package.json

```diff
@@ -9,6 +9,7 @@
     "babel-preset-es2015": "^6.22.0",
     "babel-preset-react": "^6.22.0",
     "babelify": "^7.3.0",
+    "decimal.js-light": "^2.2.0",
```

Comment:
Will we ever have a use-case where we need more precision during intermediate computation (I don't see us doing multiplication and division, only adds/subs)?  If not, I'd rather not have to ever worry about storing ridiculous floats.